src/app/features/characters/components/character-card/character-card.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="wrapper" (click)="emitCharacterID()">
+  <div class="pictureOfCharacter">
+    <img src="{{char.thumbnail.path + '.' + char.thumbnail.extension}}" alt="pictureOfCharacter">
```

Comment:
I think it is better to create a variable for this src and init it in ngOnInit, or create a similar getter

Reply:
I will create a variable in the next PR